icu4c/source/test/intltest/rbbiapts.cpp

```diff
@@ -1030,7 +1030,7 @@ void RBBIAPITest::RoundtripRule(const char *dataFile) {
     parseError.offset = 0;
     LocalUDataMemoryPointer data(udata_open(U_ICUDATA_BRKITR, "brk", dataFile, &status));
     uint32_t length;
-    const char *builtSource;
+    UnicodeString *builtSource;
```

Comment:
```suggestion
    UnicodeString builtSource;
```

Reply:
Thanks for the catch. Amended and re-pushed.